slidez-fe-core/src/components/poll/PollEditor.tsx

```diff
@@ -0,0 +1,150 @@
+                    }}
+                    render={({ values }) => (
+                        <Form>
+                            <FieldArray
+                                name='options'
+                                render={(arrayHelpers) => (
+                                    <div>
+                                        {values.options &&
+                                        values.options.length > 0 ? (
+                                            values.options.map(
+                                                (option, index) => (
+                                                    <div
+                                                        key={index}
+                                                        className='options'
+                                                    >
+                                                        <Field
+                                                            name={`options.${index}`}
+                                                            placeholder='Your option'
+                                                            className='input-options'
+                                                        />
+                                                        <button
+                                                            type='button'
+                                                            className='options-btn'
+                                                            onClick={() =>
+                                                                arrayHelpers.remove(
+                                                                    index
+                                                                )
+                                                            }
+                                                        >
+                                                            -
+                                                        </button>
+                                                        <button
+                                                            type='button'
+                                                            className='options-btn'
+                                                            onClick={() =>
+                                                                arrayHelpers.insert(
+                                                                    index,
+                                                                    ''
+                                                                )
+                                                            }
+                                                        >
+                                                            +
+                                                        </button>
+                                                    </div>
+                                                )
```

Comment:
Create a function, which will return this component part (on the top of PollEditor)